pkg/cmd/kind/root.go

```diff
@@ -69,7 +69,7 @@ func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
 		"verbosity",
 		"v",
 		0,
-		"info log verbosity",
+		"info log verbosity. higher value produces more output",
```

Comment:
```suggestion
		"info log verbosity, higher value produces more output",
```
or we start with caps after the dot or we use a comma, right?